Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -21,7 +21,7 @@ public class NoOpRunner : ISubject
     {
         public bool IsGameStarted { get; set; } = false;
 
-        public event EventHandler OnLoopFired;
+        // public event EventHandler OnLoopFired;//Un
```

Comment:
?

Reply:
Unused from the beginning of project, checked if fails to run after commenting